students/john_navitsky/session02/fizz_buzz.py

```diff
@@ -0,0 +1,21 @@
+
+""" Print fizz or buzz if the value is a multiple of 3 or five, respectively. """
+
+for i in range(1, 101):
+
+    message=""
+
+    # is the number a multiple of three?
+    if int( i / 3 ) == float( i / 3 ):
```

Comment:
you don't need the "float" the results of division are always a float.

and you don't need the "int" either, as you can use the "floor division" operator (double slash): `i // 3`

but you do'nt need any of that, as there is the "mod" or remainder operator: `%`

so this becomes:

```
if i % 3 == 0:
```

or even:

```
if not i % 3:
```
(as anything other than zero is True)